src/store/plugins/initSdk/MemoryAccount.js

```diff
@@ -0,0 +1,20 @@
+import Account from '@aeternity/aepp-sdk/es/account';
+
+async function sign() {
+  return Promise.resolve(this.signedTx);
+}
+
+async function address() {
+  return Promise.resolve(this.publicKey);
+}
+
+const MemoryAccount = Account.compose({
+  init({ publicKey, signedTx }) {
+    this.publicKey = publicKey;
+    this.signedTx = signedTx;
+  },
+  props: { publicKey: '', signedTx: '' },
+  methods: { sign, address },
+});
+
+export default MemoryAccount;
```

Comment:
Proposing to rename it to something like MemoryAccountStub, because actually it doesn't implement a fully functional Account